terraform/nr-alerts/variables.tf

```
@@ -0,0 +1,49 @@
+# NR Account ID
+variable "account_id" {
+  default = ""
+}
+
+# NR User Api Key
+variable "api_key" {
+  default = ""
+}
+
+# US/EU/Staging
+variable "region" {
+  default = ""
+}
+
+variable "instance_name_pattern" {
+  default = "canary:v0.0.0:*"
+}
```

Based on the comment I left above:
```suggestion
variable "instance_name_pattern" {
  type = object({
    name    = string
    version = string
    distro  = string
  })
  
  default = {
    name    = "canary"
    version = "v0.0.0"
    distro  = "*"
  }
}
```

---

👍  I'll give it a thought thanks!